storehaus-core/src/main/scala/com/twitter/storehaus/CollectionOps.scala

```diff
@@ -28,11 +28,12 @@ object CollectionOps {
    * PivotEncoder to Map[InnerK, V] (vs Iterable[(InnerK, V)], as
    * com.twitter.bijection.Pivot would provide).
    */
-  def pivotMap[K, OuterK, InnerK, V](pairs: Map[K, V])(split: K => (OuterK, InnerK)): Map[OuterK, Map[InnerK, V]] =
+  def pivotMap[K, OuterK, InnerK, V](pairs: Map[K, V])(
+      split: K => (OuterK, InnerK)): Map[OuterK, Map[InnerK, V]] =
     pairs.toList.map {
```

Comment:
while we are at it can we do: `pairs.iterator.map { }.toList.groupBy` which might be marginally faster.